packages/flutter/test/material/outline_button_test.dart

```diff
@@ -1043,6 +1043,97 @@ void main() {
     _checkPhysicalLayer(buttonElement, fillColor.withAlpha(0x00));
   });
 
+  testWidgets('OutlineButton respects the provided materialTapTargetSize', (WidgetTester tester) async {
+    final Key key1 = UniqueKey();
+    await tester.pumpWidget(
+      Directionality(
+        textDirection: TextDirection.ltr,
+        child: Material(
+          child: Center(
+            child: OutlineButton(
+              key: key1,
+              materialTapTargetSize: MaterialTapTargetSize.padded,
+              child: const SizedBox(width: 50.0, height: 8.0),
+              onPressed: () { },
+            ),
+          ),
+        ),
+      ),
+    );
+
+    expect(tester.getSize(find.byKey(key1)), const Size(88.0, 48.0));
```

Comment:
This is OK, but it shouldn't be necessary to use a key to identity the OutlineButton (here and elsewhere). This will work as well:
```dart
tester.getSize(find.byType(OutlineButton))
```